.changelog/2803.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:enhancement
+resource/mongodbatlas_privatelink_endpoint: Improve error message when privatelink endpoint returns error after POST
```

Comment:
```suggestion
resource/mongodbatlas_privatelink_endpoint: Improves error message when privatelink endpoint returns error after POST
```

we use 3rd person, see: https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CHANGELOG.md

Reply:
Understood, changed! 👍